ops/model.py

```diff
@@ -122,6 +125,7 @@ class Model:
     as ``self.model`` from any class that derives from :class:`Object`.
     """
 
+    @tracer.start_as_current_span('ops.Model')  # type: ignore
```

Comment:
I thought are approach was going to be "trace external things like hook tools and Pebble calls", not most methods. In this file we seem to be taking more of an "every model method" approach? Is that required to get more meaningful traces? What would it look like with just hook tools + Pebble traced?

Reply:
dropped